src/components/sprint-view-table/tasks-table.tsx

```diff
@@ -0,0 +1,160 @@
+import { Box, Card, CircularProgress, IconButton, Typography } from "@mui/material";
+import { Projects, Tasks, TimeEntries } from "../../generated/homeLambdasClient";
+import { DataGrid } from "@mui/x-data-grid";
+import { useEffect, useState } from "react";
+import { useLambdasApi } from "../../hooks/use-api";
+import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
+import KeyboardArrowDownIcon from "@mui/icons-material/KeyboardArrowDown";
+import strings from "../../localization/strings";
+import sprintViewTasksColumns from "./sprint-tasks-columns";
+import { errorAtom } from "../../atoms/error";
+import { useSetAtom } from "jotai";
+
+/**
+ * Component properties
+ */
+interface Props {
+  project : Projects,
+  loggedInpersonId?: number,
+  filter?: string
+}
+
+/**
+ * Task component
+ * 
+ * @param props component properties
+ */
+const TaskTable = ({project, loggedInpersonId, filter}: Props) => {
+  const { tasksApi, timeEntriesApi } = useLambdasApi();
+  const [tasks, setTasks] = useState<Tasks[]>([]);
+  const [timeEntries, setTimeEntries] = useState<number[]>([]);
+  const [open, setOpen] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const columns = sprintViewTasksColumns({tasks, timeEntries});
+  const setError = useSetAtom(errorAtom);
+  const [reload, setReload] = useState(false);
```

Comment:
I'm very suspicious of this, are you sure its needed? It seems to be tied to the loggedInPersonId changing, but when do we expect that to happen? 
It could be worth us discussing this together to figure out if it can be removed/ replaced with something we already have.

Reply:
We give parameter loggedInPersonId to task table component: if it is empty then we get all tasks, else we get specific person tasks. We modified the logic so that now if time entries firstly requested for a person they are not re-requested again. We added reload to fix problem when changes in loggedInPerson force re-requesting of tasks and time-entries.